code/game/gamemodes/cult/runes.dm

```diff
@@ -855,7 +870,7 @@ var/list/sacrificed = list()
 		if(istype(cultist.loc, /obj/machinery/dna_scannernew)&&cultist.loc:locked)
 			cultist.loc:locked = 0
 		for(var/mob/living/carbon/C in users)
-			user.take_overall_damage(dam, 0)
+			C.rune_apply_damage(10, 15)

```

Comment:
Move this line to _before_ the for loop. Change it to `user.apply_damage(10, 15)`